vector/src/main/res/layout/fragment_ftue_display_name.xml

```diff
@@ -0,0 +1,151 @@
+            app:layout_constraintBottom_toTopOf="@id/displayNameHeaderTitle"
+            app:layout_constraintEnd_toEndOf="@id/displayNameGutterEnd"
+            app:layout_constraintHeight_percent="0.15"
+            app:layout_constraintStart_toStartOf="@id/displayNameGutterStart"
+            app:layout_constraintTop_toTopOf="parent"
+            app:tint="?colorSecondary" />
+
+        <TextView
+            android:id="@+id/displayNameHeaderTitle"
+            style="@style/Widget.Vector.TextView.Title.Medium"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="16dp"
+            android:gravity="center"
+            android:text="@string/ftue_display_name_title"
+            android:textColor="?vctr_content_primary"
+            app:layout_constraintBottom_toTopOf="@id/displayNameHeaderSubtitle"
+            app:layout_constraintEnd_toEndOf="@id/displayNameGutterEnd"
+            app:layout_constraintStart_toStartOf="@id/displayNameGutterStart"
+            app:layout_constraintTop_toBottomOf="@id/displayNameHeaderIcon" />
+
+        <TextView
+            android:id="@+id/displayNameHeaderSubtitle"
+            style="@style/Widget.Vector.TextView.Subtitle"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="8dp"
+            android:gravity="center"
+            android:text="@string/ftue_display_name_subtitle"
+            android:textColor="?vctr_content_secondary"
+            app:layout_constraintBottom_toTopOf="@id/titleContentSpacing"
+            app:layout_constraintEnd_toEndOf="@id/displayNameGutterEnd"
+            app:layout_constraintStart_toStartOf="@id/displayNameGutterStart"
+            app:layout_constraintTop_toBottomOf="@id/displayNameHeaderTitle" />
+
+        <Space
+            android:id="@+id/titleContentSpacing"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            app:layout_constraintBottom_toTopOf="@id/displayNameInput"
+            app:layout_constraintHeight_percent="0.03"
+            app:layout_constraintTop_toBottomOf="@id/displayNameHeaderSubtitle" />
+
+        <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/displayNameInput"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:hint="@string/ftue_display_name_entry_title"
+            app:layout_constraintEnd_toEndOf="@id/displayNameGutterEnd"
+            app:layout_constraintStart_toStartOf="@id/displayNameGutterStart"
+            app:layout_constraintTop_toBottomOf="@id/titleContentSpacing">
+
+            <com.google.android.material.textfield.TextInputEditText
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:maxLines="1" />
```

Comment:
Could be nice to set an inputType and `android:imeOptions="actionDone"`

Reply:
great idea, will do :+1:

Reply:
added 6fb5d05dc561bf54dcbfb2f8d901a8a3e894db91